src/test/java/com/lob/model/USAutocompletionTest.java

```diff
@@ -0,0 +1,28 @@
+package com.lob.model;
+
+import com.lob.BaseTest;
+import com.lob.net.LobResponse;
+import org.junit.Test;
+
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.junit.Assert.*;
+
+public class USAutocompletionTest extends BaseTest {
+
+    @Test
+    public void testUSAutocompletion() throws Exception {
+        LobResponse<USAutocompletion> response = new USAutocompletion.RequestBuilder()
+                .setAddressPrefix("185 BER")
```

Comment:
Nit: Did you over indent?

Reply:
nvm just gonna change it

Reply:
I don't know what's the reasoning behind 2 indents instead of 1, but I've never seen that. It's not python, so number of indents doesn't matter towards functionality. It's simply a stylistic preference, which mine would be to use 2 spaces instead of 4, but that's a different topic of discussion.